mod-audit-server/src/main/java/org/folio/services/OrderAuditEventService.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.services;
+
+import io.vertx.core.Future;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import org.folio.rest.jaxrs.model.OrderAuditEvent;
+
+public interface OrderAuditEventService {
+  Future<RowSet<Row>> collectData(OrderAuditEvent orderAuditEvent, String tenantId);
```

Comment:
Please rename this method, also make similar changes as for dao OrderAuditEventsService and introduce acuqisition package, because this module also will be used by other teams from different domains